web/Dockerfile

```diff
@@ -42,16 +42,16 @@ RUN npm run build
 
 ################################################################################
 
-FROM node:12-alpine
+FROM registry.fedoraproject.org/fedora:33
+
+RUN dnf update -y && dnf install -y nodejs && dnf clean all
```

Comment:
We could probably remove this and move the dnf clean all line 20.

Reply:
How node is going to be installed in the final image? Cleaning builder base image does not seem necessary.

Reply:
Ok I missed about the multi-stage build system. Ok good so let's merge it ! thanks !